skfair/metrics/fairness_report.py

```diff
@@ -0,0 +1,117 @@
+from functools import partial
+from collections import defaultdict
+
+from sklearn.metrics import (confusion_matrix, f1_score, precision_score,
+                             accuracy_score, recall_score)
+import numpy as np
+import pandas as pd
+
+
+def false_positive_score(y_true, y_pred, labels=None):
+    conf_matrix = confusion_matrix(y_true, y_pred, labels=labels)
+    tn, fp, fn, tp = true_false_positive_negative(conf_matrix)
+    eps = 1e-10
+    return fp / (fp + tn + eps)
+
+
+def false_discovery_score(y_true, y_pred, labels=None):
+    conf_matrix = confusion_matrix(y_true, y_pred, labels=labels)
+    tn, fp, fn, tp = true_false_positive_negative(conf_matrix)
+    eps = 1e-10
+    return fp / (tp + fp + eps)
```

Comment:
These are nice to have. Preferably we'd have these metrics in a seperate file though. You could also add a unit test/docstring but that can also be left for another PR.

Reply:
👍